apps/app/src/app/pages/(components)/components/(breadcrumb)/breadcrumb.page.ts

```diff
@@ -0,0 +1,143 @@
+		CodePreviewDirective,
+		TabsCliComponent,
+
+		BreadcrumbPreviewComponent,
+		BreadcrumbCustomSeparatorComponent,
+		BreadcrumbDropdownComponent,
+		BreadcrumbCollapsedComponent,
+	],
+	template: `
+		<section spartanMainSection>
+			<spartan-section-intro
+				name="Breadcrumb"
+				lead="Displays the path to the current resource using a hierarchy of links."
+			/>
+			<spartan-tabs firstTab="Preview" secondTab="Code">
+				<div spartanCodePreview firstTab>
+					<spartan-breadcrumb-preview />
+				</div>
+				<spartan-code secondTab [code]="defaultCode" />
+			</spartan-tabs>
+
+			<spartan-section-sub-heading id="installation">Installation</spartan-section-sub-heading>
+			<spartan-cli-tabs
+				class="mt-4"
+				nxCode="npx nx g @spartan-ng/cli:ui breadcrumb"
+				ngCode="ng g @spartan-ng/cli:ui breadcrumb"
+			/>
+
+			<spartan-section-sub-heading id="usage">Usage</spartan-section-sub-heading>
+			<div class="space-y-4">
+				<spartan-code [code]="defaultImports" />
+				<spartan-code [code]="defaultSkeleton" />
+			</div>
+
+			<spartan-section-sub-heading id="examples">Examples</spartan-section-sub-heading>
+			<h3 id="examples__default" class="${hlmH4} mt-6">Custom separator</h3>
+			<p class="${hlmP} mb-2">
+				Use a custom component as
+				<code class="${hlmCode} mr-0.5">children</code>
+				for
+				<code class="${hlmCode} mr-0.5">HlmBreadcrumbSeparator</code>
+				to create a custom separator.
+			</p>
+			<spartan-tabs firstTab="Preview" secondTab="Code">
+				<div spartanCodePreview firstTab>
+					<spartan-breadcrumb-custom-separator />
+				</div>
+				<spartan-code secondTab [code]="customSeparatorCode" />
+			</spartan-tabs>
+			<div class="mt-6">
+				<spartan-code [code]="customSeparatorSkeleton" />
+			</div>
+
+			<hr class="my-4 md:my-8" />
+
+			<h3 id="examples__default" class="${hlmH4}">Dropdown</h3>
+			<p class="${hlmP} mb-2">
+				You can compose
+				<code class="${hlmCode} mr-0.5">HlmBreadcrumbItem</code>
+				for
+				<code class="${hlmCode} mr-0.5">HlmDropdownMenu</code>
+				to create a custom separator.
+			</p>
+			<spartan-tabs firstTab="Preview" secondTab="Code">
+				<div spartanCodePreview firstTab>
+					<spartan-breadcrumb-dropdown />
+				</div>
+				<spartan-code secondTab [code]="dropdownCode" />
+			</spartan-tabs>
+			<div class="mt-6">
+				<spartan-code [code]="dropdownSkeleton" />
+			</div>
+
+			<hr class="my-4 md:my-8" />
+
+			<h3 id="examples__default" class="${hlmH4}">Collapsed</h3>
+			<p class="${hlmP} mb-2">
+				We provide a
+				<code class="${hlmCode} mr-0.5">BreadcrumbEllipsis</code>
+				component to show a collapsed state when the breadcrumb is too long.
+			</p>
+			<spartan-tabs firstTab="Preview" secondTab="Code">
+				<div spartanCodePreview firstTab>
+					<spartan-breadcrumb-collapsed />
+				</div>
+				<spartan-code secondTab [code]="collapsedCode" />
+			</spartan-tabs>
+			<div class="mt-6">
+				<spartan-code [code]="collapsedSkeleton" />
+			</div>
+		</section>
```

Comment:
Add missing bottom navigation and table of content to the breadcrumb component page. Also change the bottom navigation for `button` and `card` to point to `breadcrumb`.

```suggestion
         <spartan-page-bottom-nav>
				<spartan-page-bottom-nav-link href="card" label="Card" />
				<spartan-page-bottom-nav-link direction="previous" href="button" label="Button" />
			</spartan-page-bottom-nav>   
		</section>
		<spartan-page-nav />
```

Reply:
I added it. 
Thanks

Reply:
Missing `<spartan-page-nav />` below the section for the table of content on that page.

Reply:
I added it. Thanks

Reply:
Please update "badge" to point to "breadcrumb" instead of "button"

![CleanShot 2024-11-19 at 10 55 50](https://github.com/user-attachments/assets/a1fd7d0c-143d-4cbd-8f70-c996c705808f)

Reply:
I fixed it